docs/ops/sequence/OneHot_1.md

```diff
@@ -32,35 +32,66 @@ Types of input scalars `on_value` and `off_value` should match and can be any of
 
 **Inputs**:
 
-* **1**: `indices`: input tensor of rank `N` with indices of any supported integer data type. Can be 0D. Required.
-* **2**: `depth`: scalar (0D tensor) of any supported integer type that specifies number of classes and the size of one-hot dimension.
-* **3**: `on_value`: scalar (0D tensor) of any type that is the value that the locations in output tensor represented by indices in input take.
-* **4**: `off_value`: scalar (0D tensor) of any type that is the value that the locations not represented by indices in input take.
+* **1**: `indices`: input tensor of type T1 with non-negative indices, behavior for negative indices is undefined. Can be 0D. Required.
```

Comment:
Wrap types "T1" and "T2" with asterisk character like this: `...of type *T1* with...`

Reply:
Corrected.